### PR TITLE
update install procedure and prerequisite.

### PR DESCRIPTION
**Pull request by fujitatomoya**
minor fix to address https://github.com/ros-infrastructure/rosdoc2/issues/66

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,18 +11,21 @@ This tool can be viewed from two perspectives: first from the perspective of a u
 To generate the documentation for almost any ROS 2 package, first build the package locally, then run this command replacing the arguments with appropriate directories:
 
 ```
-$ rosdoc2 build \
+rosdoc2 build \
```
Comment (fujitatomoya) [inline README.md:14]: `$ ` is in the way to do `copy&paste`.

**Comment by fujitatomoya** [comment]
@Yadunund @audrow can you take a look?

**Review by Yadunund — APPROVED** [review]
LGTM! Thanks for the improving the instructions here.

File: README.md [hunk 2]
```diff
@@ -11,18 +11,21 @@ This tool can be viewed from two perspectives: first from the perspective of a u
...
   --package-path ./src/path/to/my_package_name
 ```
 
+> [!NOTE]
+> Please make sure that ROS 2 colcon workspace is NOT sourced during build.
```
Comment (Yadunund) [inline README.md:19]: On second thought I'm not sure if we should make this generalization for reasons explained here https://github.com/ros-infrastructure/rosdoc2/issues/66#issuecomment-2001897492.

How about we create a `Known Issues` section at the bottom of the README and link https://github.com/ros-infrastructure/rosdoc2/issues/68?

**Review by Yadunund — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fujitatomoya) [reply README.md:19]: sgtm.

**Comment by fujitatomoya** [comment]
I think `Known Issues` are expected to be fixed in the future, because those are issues. so technically we do not want to have this section at 1st place, but i believe that at least we can do this for now. (i am not really sure why this problem is happening.)

CC: @Yadunund

**Pull request merged**